src/full/Agda/TypeChecking/Serialise/Instances/Common.hs

```diff
@@ -314,6 +285,21 @@ instance EmbPrj a => EmbPrj (P.Interval' a) where
 
   value = valueN P.Interval
 
+instance EmbPrj RangeFile where
+  icod_ (RangeFile _ Nothing)  = __IMPOSSIBLE__
+  icod_ (RangeFile _ (Just a)) = icode a
+
+  value r = do
+    m :: TopLevelModuleName
+            <- value r
+    mf      <- gets modFile
+    incs    <- gets includes
+    (r, mf) <- liftIO $ findFile'' incs m mf
```

Comment:
Do we deserialize `RangeFile` often?

Reply:
No, see a87578f1c874a4b0498fcc2f078db8602d438c12.